docs/source/using.rst

```diff
@@ -89,3 +89,36 @@ When Voilà is run on a notebook, the following steps occur:
    Jupyter server.
 #. When users access the page, the widgets on the page have access to
    the underlying Jupyter kernel.
+
+The example notebooks
+=====================
+
+The `notebooks directory <https://github.com/voila-dashboards/voila/tree/master/notebooks>`__
+contains a collection of Jupyter notebooks that can be rendered using Voilà:
+
+* **basics.ipynb** - a notebook with interactions requiring a roundtrip to the kernel.
+* **bqplot.ipynb** - uses custom Jupyter widgets such as
+  `bqplot <https://github.com/bloomberg/bqplot>`__.
+* **dashboard.ipynb** - uses gridstack.js for the layout of each output.
```

Comment:
We could also include the new `gridspecLayout.ipynb` from https://github.com/voila-dashboards/voila/pull/452.

Reply:
Thanks @jtpio :sunflower: